plugins/oderave/CMakeLists.txt

```diff
@@ -50,17 +50,14 @@ if(ODE_LIBRARY_FOUND)
 
   message(STATUS "ODE found, building oderave plugin, precision=${ODE_PRECISION}")
 
-  target_link_libraries(oderave libopenrave ${ODE_LIBRARY})
-  target_link_libraries(oderave PRIVATE boost_assertion_failed)
+  target_link_libraries(oderave PUBLIC libopenrave ${ODE_LIBRARY} PRIVATE boost_assertion_failed)
   set_target_properties(oderave PROPERTIES COMPILE_FLAGS "${PLUGIN_COMPILE_FLAGS} ${ODE_CXXFLAGS}" LINK_FLAGS "${PLUGIN_LINK_FLAGS}")
   install(TARGETS oderave DESTINATION ${OPENRAVE_PLUGINS_INSTALL_DIR} COMPONENT ${COMPONENT_PREFIX}plugin-oderave)
   
   if( MSVC )
     # have to install the DLLs
     install(DIRECTORY "${ODE_LINK_DIRS}/" DESTINATION bin COMPONENT ${COMPONENT_PREFIX}plugin-oderave FILES_MATCHING PATTERN "*-${MSVC_PREFIX}-*.dll")
   endif()
-else()
-  message(STATUS "Could not find ODE. Please install ODE (http://www.ode.org)")
```

Comment:
is removing this ok?

Reply:
That message is redundant, the same message is printed if FindODE.cmake fails to find ODE.
Message is printed twice so I removed one.